pkgs/applications/window-managers/gamescope/default.nix

```diff
@@ -102,15 +110,18 @@ stdenv.mkDerivation {
     stb
     hwdata
     openvr
-    vkroots
     libdisplay-info
   ];
 
   outputs = [ "out" "lib" ];
 
   postUnpack = ''
     rm -rf source/subprojects/vkroots
-    ln -s ${vkroots} source/subprojects/vkroots
+    ln -s ${finalAttrs.vkroots} source/subprojects/vkroots
```

Comment:
Should this be labeled as a `# HACK`? Maybe put in the same comment from the description about possible upstream improvements?